CHANGELOG.md

```diff
@@ -0,0 +1,45 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+### Added
+
+* Setting options for the plugin using options of the TinyMCE editor.
+
+### Changed
+
+* Bumped the version of the widget to `3.x`, see [widget changelog][widget changelog].
```

Comment:
Bumped the widget version to `3.x`, see `[widget changelog][widget-changelog]` for details.

Note: our naming convention for link references implies using dashes instead of spaces.